change/@microsoft-fast-ssr-e107db57-9534-4ac4-bd65-157d5b40a79f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "After install DOM Shim, globals will now return the original global instead of the shimmed version in async local storage if code is not in storage scope.",
+  "packageName": "@microsoft/fast-ssr",
+  "email": "erhuan@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
This should also be prerelease

Reply:
I did select prerelease under the `yarn change` prompt. Can I just hand edit this file?

Reply:
Yes, you can just hand edit that and commit. It's a bit of a quick of beachball for prerelease packages.

Reply:
Resolved now